next.config.js

```diff
@@ -1,6 +1,12 @@
 /** @type {import('next').NextConfig} */
 
-const nextConfig = {};
+const nextConfig = {
+  reactStrictMode: true,
+  webpack: (config) => {
+    config.externals = [...config.externals, 'canvas', 'jsdom', 'node-fetch'];
```

Comment:
??

Reply:
Required by isomorphic dompurify

Reply:
https://github.com/kkomelin/isomorphic-dompurify/issues/54. It will work without the above code; however it takes care of the errors in the console.